src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -53,14 +55,27 @@
+			myTests !== null
 	);
+
+	function selectMyTestsFilter(value: string) {
+		const url = new URL(page.url);
+		if (value === 'all') {
+			url.searchParams.delete('my_tests');
+		} else {
+			url.searchParams.set('my_tests', value);
+		}
+		url.searchParams.set('page', '1');
+		goto(url, { keepFocus: true, invalidateAll: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)